media/static-bundles.json

```diff
@@ -1042,6 +1036,24 @@
       ],
       "name": "firefox_whatsnew_70_experiment_syncbutton"
     },
+    {
+      "files": [
+        "js/base/uitour-lib.js",
+        "js/firefox/whatsnew/up-to-date.js",
+        "js/base/mozilla-fxa-product-button.js",
+        "js/base/mozilla-fxa-product-button-init.js"
+      ],
+      "name": "firefox_whatsnew_71"
```

Comment:
question: should we consider content to show to users who have JS disabled? I know this is a small %, but considering that this page is shown to all users it might be worth having a cta fallback for them.

Reply:
The WNP itself is fully accessible for users with JS disabled. I don't think we can go much further here at this time, seeing as it's the primary offer on the page. I would like to see the accounts.firefox.com site handle this better though.